site2/docs/concepts-messaging.md

```diff
@@ -906,7 +906,7 @@ Message deduplication is disabled in the scenario shown at the top. Here, a prod
 In the second scenario at the bottom, the producer publishes message 1, which is received by the broker and persisted, as in the first scenario. When the producer attempts to publish the message again, however, the broker knows that it has already seen message 1 and thus does not persist the message.
 
 > Message deduplication is handled at the namespace level or the topic level. For more instructions, see the [message deduplication cookbook](cookbooks-deduplication).
-
+> You can read the design of Message Deduplication in [PIP-6](https://github.com/aahmed-se/pulsar-wiki/blob/master/PIP-6:-Guaranteed-Message-Deduplication.md), and a get an overview in this [blog post](https://www.splunk.com/en_us/blog/it/effectively-once-semantics-in-apache-pulsar.html)
```

Comment:
To keep neutral and not to be challenged by ASF, we avoid adding blogs of commercial companies to the Pulsar documentation.